nss-day-cohort-46/rare-client-tragic-trolls
# Nc new post form

# Description

The create post form will create a new form and put it in the database.

# Testing Instructions

```
ttb nc-newPostForm
```

the nav bar should contain "New Post". click on it.

Fill out the form.

Click submit.

Check your database to see if a new form entry is there and that your tag join row was put into the database.
